app/views/records/show_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
 <% record.rights.each do |r| %>
-  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%> <%= iconify_auto_link(r,false) %><br />
+  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%><% if is_url?(r) %> <%=iconify_auto_link(r,false) %><% end %><br />
```

Comment:
Can we put this code into the `iconify_auto_link` helper method instead of the view?